src/app/zap-templates/templates/chip/encoder-src.zapt

```diff
@@ -193,15 +193,29 @@ PacketBufferHandle encode{{asCamelCased parent.name false}}ClusterRead{{asCamelC
 }
 
 {{#if (isWritableAttribute)}}
-PacketBufferHandle encode{{asCamelCased parent.name false}}ClusterWrite{{asCamelCased name false}}Attribute(uint8_t seqNum, EndpointId destinationEndpoint, {{asUnderlyingZclType type}} {{asCamelCased name}})
+PacketBufferHandle encode{{asCamelCased parent.name false}}ClusterWrite{{asCamelCased name false}}Attribute(uint8_t seqNum, EndpointId destinationEndpoint, {{chipType}} {{asCamelCased name}})
 {
     COMMAND_HEADER("Write{{asCamelCased parent.name false}}{{asCamelCased name false}}", {{parent.define}}_ID);
+    {{#if (isString type)}}
```

Comment:
This definitely looks like it needs a bytestring case, like the one command encoding has.